### DISCUSSION TITLE
Advanced smart cropping not working

### DISCUSSION THREAD

**Discussion by marcuslindblom** [imgproxy/imgproxy#1145] [Q&A] [answered]
I'm using the following config with [this source](https://live.staticflickr.com/173/422892014_b1db80a824_o.jpg) but the smart cropping feature does not seem to be enabled. 

Config:
![image](https://user-images.githubusercontent.com/319720/229106019-0eb2e381-2685-46c9-a7ab-165bdf0354d6.png)

This is the restult using `/w:150/h:150/gravity:sm/rt:fill` using `avif` as format.
![image](https://user-images.githubusercontent.com/319720/229106309-01ccc2b3-0280-4de7-aa94-9438283b9146.png)

**Comment by DarthSim** [comment, marked as answer]
Found the case where face detection may not be applied. Fixed in the `latest` build

**Reply by marcuslindblom** [reply]
It seems to work fine now. Thanks
